### ISSUE TITLE
Issue ID:  DAC_Custom_Accordions1 - Feature: Q&A atom

### ISSUE THREAD

**Issue by VDuczekW** [guardian/dotcom-rendering#4446]
see report: https://drive.google.com/file/d/1qAxySwEa2BY0JY_Le9I7c4mp_Mr4zOum/view

**Comment by SiAdcock** [comment]
Details / summary might not be inherently accessible across all browser /  screen reader combinations. Scott O'Hara provides [insight into best practice](https://www.scottohara.me/blog/2018/09/03/details-and-summary.html), which boils down to:

```
<details>
  <summary aria-expanded="true/false" tabindex="0" role="button">
    My trigger
  </summary>
  <div>
    <!-- content goes here -->
  </div>
</details>
```

**Comment by SiAdcock** [comment]
The Voice Activation issue is clear to us, and it would be useful to discuss workarounds for it.

We are unable to reliably recreate the screen reader issues documented in the report. We would find it useful to watch a screen reader user using this control so that we are able to design a better experience.

**Comment by mxdvl** [comment]
There is now a Source component in the kitchen that seems to do this, but it does not use `details`/`summary` since https://github.com/guardian/source/pull/1615 : [`ExpandingWrapper.tsx`](https://github.com/guardian/csnx/tree/5a978e896eef1649cdbcbacc9637039c9db55713/libs/%40guardian/source-react-components-development-kitchen/src/expanding-wrapper)

**Comment by bryophyta** [comment]
there's also an updated version of the article that Simon linked to above now: https://www.scottohara.me//blog/2022/09/12/details-summary.html

**Comment by VDuczekW** [comment]
expandable wrapper done by Sophie McMillan - check in on that piece - moving to should as not being worked on and no way forward understood
